### ISSUE TITLE
CVE-2017-13098

### ISSUE THREAD

**Issue by karlhe** [jruby/jruby-openssl#156]
There's a TLS vulnerability fixed in bouncy castle 1.59: https://www.kb.cert.org/vuls/id/CHEU-AT5U7K

I'm assuming it would've been updated in the next jruby-openssl release regardless, but dropping a note here.

**Comment by headius** [comment]
Hey, thanks for opening the issue! We'll get a release spun as soon as possible. BC has an annoying tendency to break their own APIs.

If you're so inclined, you might try updating it yourself (follow build instructions in the repo) and see if what you run into :-)

**Comment by kares** [comment]
although we would love to have the resources to switch to BC's JSSE - making SSL/TLS more predictable.
we're using much of BC but not its SSL engine - so far it isn't even included in any of the jossl releases ...
(included in master as hopes for starting to use it but that wasn't released and is blocked by another issue)
